recipes/taocpp-json/config.yml

```diff
@@ -1,3 +1,5 @@
 versions:
   "1.0.0-beta.11":
     folder: all
+  "1.0.0-beta.10":
```

Comment:
Why would somebody use `1.0.0-beta.10` if `1.0.0-beta.11` is available?
All of these are betas btw. Not final releases.